SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/TVA/SOE.tsx

```diff
@@ -57,17 +57,20 @@ const SOE: React.FC<SEBrowser.IWidget<any>> = (props) => {
     }
 
     function HandleStatusFilterChange(key: string) {
-        statusFilter[key] = !statusFilter[key]
+        const lowercaseKey = key.toLowerCase();
```

Comment:
switch typing of key to be `keyof ISOEFilter` and define `interface ISOEFilter = { abnormal: boolean, .....}` that will also take care of cxasing if you change type of `statusFilter`

Reply:
This has been done.